src/style.scss

```diff
@@ -224,3 +224,18 @@ textarea {
     color: var(--header-bg);
   }
 }
+
+.block-container {
+  height: 250px;
+  margin-bottom: 24px !important;
+}
+
+.no-space {
```

Comment:
We don't need a custom class here you can use the class "p-4" and "text-align" instead.